src/test/java/com/github/javafaker/DragonBallTest.java

```diff
@@ -0,0 +1,13 @@
+package com.github.javafaker;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+
+public class DragonBallTest extends AbstractFakerTest {
```

Comment:
Any chance you could use hamcrest matches and also put in assertions that do more than just asserting on whether something is empty or not, like maybe a regex. You can see how the other tests are written.